docs/source/reference/numpysupported.rst

```diff
@@ -264,6 +264,7 @@ The following top-level functions are supported:
 * :func:`numpy.concatenate`
 * :func:`numpy.convolve` (only the 2 first arguments)
 * :func:`numpy.copy` (only the first argument)
+* :func:`numpy.corrcoef` (only the 3 first arguments, requires NumPy >= 1.10 and SciPy >= 0.16; extreme value handling per NumPy 1.11+)
```

Comment:
Tiny thing... Any chance you could please put a line break in as this is a really long line now. I think the syntax is to line up the broken line with the first `:` of `:func` to get it to be a continued list item. Thanks.

Reply:
Done - hopefully correctly (I have never had much luck building the docs).  Let me know if it's OK, cheers!

Reply:
Fix looks fine on inspection. Thanks.